player/minimax/minimax_player.py

```diff
@@ -0,0 +1,69 @@
+import math
+
+import numpy as np
+
+from game_logic.game_status import GameStatus
+from game_logic.igame_state import IGameState
+from player.iplayer import IPlayer
+from player.minimax.minimax_configuration import MinimaxConfiguration
+
+
+class MinimaxPlayer(IPlayer):
+
+    def __init__(self, minimax_configuration: MinimaxConfiguration):
+        self.minimax_configuration = minimax_configuration
+        self._heuristic_fields_values = np.array([
+                [3, 4,  5,  7,  5, 4, 3],
+                [4, 6,  8, 10,  8, 6, 4],
+                [5, 8, 11, 13, 11, 8, 5],
+                [5, 8, 11, 13, 11, 8, 5],
+                [4, 6,  8, 10,  8, 6, 4],
+                [3, 4,  5,  7,  5, 4, 3]
+            ])
+
+    # TODO:
+    #  If its hardcoded, it cannot be used for other board sizes, maybe it should be generated?
+    @property
+    def heuristic_fields_values(self):
+        return self._heuristic_fields_values
+
+    def get_and_make_next_move_for_player(self, game_state: IGameState) -> int:
+        game_state_copy = game_state.clone()
+        is_first_player = game_state.is_first_player_move
+        evaluation, move = self.minimax(game_state_copy, 0, True, None, is_first_player)
```

Comment:
could we add named arguments here? like current_depth=0 etc.?

Reply:
![image](https://github.com/Michalski-Artur/MCTS-Connect-4/assets/79529984/0864d4d2-c9af-4f9c-87dc-adb222946a0c)

Reply:
Added!